oshi-core/src/main/java/oshi/driver/unix/openbsd/disk/Disklabel.java

```diff
@@ -0,0 +1,127 @@
+/**
+ * MIT License
+ *
+ * Copyright (c) 2010 - 2020 The OSHI Project Contributors: https://github.com/oshi/oshi/graphs/contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package oshi.driver.unix.openbsd.disk;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.ListIterator;
+import java.util.Map;
+import oshi.hardware.HWPartition;
+import oshi.util.Constants;
+import oshi.util.ExecutingCommand;
+import oshi.util.ParseUtil;
+import oshi.util.tuples.Pair;
+
+public final class Disklabel {
+
+    private Disklabel() {
+    }
+
+    /*
+    * └─ $ ▶ doas disklabel sd1
+doas (mprins@thinkpad.local) password:
+# /dev/rsd1c:
+type: SCSI
+disk: SCSI disk
+label: Storage Device
+duid: 0000000000000000
+flags:
+bytes/sector: 512
+sectors/track: 63
+tracks/cylinder: 255
+sectors/cylinder: 16065
+cylinders: 976
+total sectors: 15693824
+boundstart: 0
+boundend: 15693824
+drivedata: 0
+
+*/
+    public static Pair<Map<String, String>, List<HWPartition>> getDiskParams(String diskName) {
+        List<HWPartition> partitions = new ArrayList<>();
+        HashMap<String, String> params = new HashMap<>();
+        Pair<Map<String, String>, List<HWPartition>> pair = new Pair<>(params, partitions);
+        // disklabel (requires root) supports 15 configurable partitions, `a' through `p', excluding `c'.
+        // The `c' partition describes the entire physical disk.
+        // By convention, the `a' partition of the boot disk is the root
+        // partition, and the `b' partition of the boot disk is the swap partition
+        // List<String> cmd = ExecutingCommand.runNative("doas disklabel -n " + diskName);
+        List<String> cmd = ExecutingCommand.runNative("disklabel -n " + diskName);
+        if (cmd.size() > 0) {
```

Comment:
`!cmd.isEmpty()` is preferred, although you can usually just iterate over the list and if it's empty nothing will happen inside the list.